pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources.nix

```
@@ -259,9 +259,9 @@
   };
 
   hcl = {
-    version = "1.1.0";
+    version = "1.2.0";
     url = "github:MichaHoffmann/tree-sitter-hcl";
```

Repo has moved. Redirect points to:
```suggestion
    url = "github:tree-sitter-grammars/tree-sitter-hcl";
```

---

Yep.